### PR TITLE
[actionlib] update upstream branch

### PR DESCRIPTION
**Pull request by seanyen**
Recently there are some check-ins on `indigo-devel` not compatible with the `melodic-devel` of `realtime_tools` and it causes build break if everything is built from the upstream. I would recommend to use `melodic-devel` instead as the `actionlib` upstream branch.

### CODE REVIEW

**Comment by seanyen** [comment]
cc @sloretz

**Review by clalancette — APPROVED** [review]
This seems reasonable to me, but I'll let @mjcarroll review and merge this one since he is the maintainer of actionlib.

**Review by sloretz — APPROVED** [review]

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**
